Documentation/contributor-guide/logging.md

```diff
@@ -1,6 +1,6 @@
 # Logging Conventions
 
-etcd uses the [capnslog][capnslog] library for logging application output categorized into *levels*. A log message's level is determined according to these conventions:
+etcd uses the [zap][zap] library for logging application output categorized into *levels*. A log message's level is determined according to these conventions:
```

Comment:
Good catch. We should also remove the contentions mentioned here, they are from capnslog. With Zap, etcd supports `debug, info, warn, error, panic, fatal. We can just list them without any details as the types are self-explanatory. Optionally, add a sentence something like "refer to the https://github.com/uber-go/zap/blob/master/logger.go to learn more about these conventions".

Reply:
ok, let me send a separate PR for that.

Reply:
Ok, thanks! That's fine. I will send it after this merge as you are in mid of many other things.